### ISSUE TITLE
Some pods don't have button to download, autorefresh logs

### ISSUE THREAD

**Issue by botzill** [kubernetes/dashboard#3754]
##### Environment

Create a cluster on DigitaOcean and install dashboard using helm chart
```
Installation method: helm chart
Kubernetes version: 1.13.5
Dashboard version: v1.10.1
Operating system: DigitalOcean K8S
Node.js version ('node --version' output):
Go version ('go version' output):
```

##### Steps to reproduce

Create a custom docker image deploy it using helm chart. Maybe docker image is made specifically so that I don't see these options.  

##### Observed result

Create a custom docker image deploy it using  helm chart

<img width="1029" alt="Screen Shot 2019-04-23 at 20 50 26" src="https://user-images.githubusercontent.com/4686040/56603956-88f4e380-6609-11e9-96d2-886a04213a32.png">

<img width="1014" alt="Screen Shot 2019-04-23 at 20 51 51" src="https://user-images.githubusercontent.com/4686040/56604009-aaee6600-6609-11e9-87cd-b03ed7db241c.png">

##### Expected result

Show option to autorefresh and download logs. Currently, I see this 

Example:

<img width="1034" alt="Screen Shot 2019-04-23 at 20 53 12" src="https://user-images.githubusercontent.com/4686040/56604084-d8d3aa80-6609-11e9-8d05-ad453a73f723.png">

**Comment by jeefy** [comment]
Thanks @botzill! 

I'm thinking that the buttons disappear if the width of the pod/container text is too wide.

/good-first-issue

**Comment by botzill** [comment]
Hi @jeefy and thx!

Yes, this issue is this. Can this be solved somehow? 

Thx.

**Comment by maciaszczykm** [comment]
@botzill It can, but at the moment we are focused on other things. If you are willing to change some CSS or HTML then feel free to work on it.

**Comment by feloy** [comment]
I think this issue were fixed by #3766
